packages/core/src/exports.ts

```diff
@@ -318,3 +322,86 @@ export function getClient<C extends Client>(): C | undefined {
 export function getCurrentScope(): Scope {
   return getCurrentHub().getScope();
 }
+
+/**
+ * Start a session on the current isolation scope.
+ *
+ * @param context (optional) additional properties to be applied to the returned session object
+ *
+ * @returns the new active session
+ */
+export function startSession(context?: SessionContext): Session {
+  const client = getClient();
+  const isolationScope = getIsolationScope();
+
+  const { release, environment = DEFAULT_ENVIRONMENT } = (client && client.getOptions()) || {};
```

Comment:
l: should we maybe just bail out if no client is found? Doesn't matter probably, but for clarity...

Reply:
Hmm thinking about this, if we do so, we have to return `undefined` which I'm not sure we want to do. Since we're only taking env and release from the client I think it's okay to continue. Obviously, on session will be sent but to me this aligns more with the concept of no-op spans for example.

Reply:
ok, sounds good to me!